script/Arcueid/Day 1/01_00_ARC01_3_2.txt

```diff
@@ -114,7 +114,7 @@ Then, I was involved in an accident that left me with a physical defect, and Mak
 [sha:bd4b56e65eae6611eb750efe9177bc37c5e80fd3]{
 -- Page 219, Offset 1734.
 -- 曰く、“たとえ長男であろうと、いつ死ぬか分からない軟弱者を後継者にはできない”とのこと。
-He said, 'Even if you're the eldest son, you can't be the successor as a weak man who might die at any time.'
+He said, 'Even if you're my eldest son, a weak man who could drop dead at any time like you cannot be my successor.'
```

Comment:
would revert the first my, 'the' comes off as more detached which I feel is the intent.

Reply:
I'd also reorder it slightly - 
> He said, 'Even if you're the eldest son, a weak man like you who could drop dead at any time cannot be my successor.'